### ISSUE TITLE
Add location and source to map tooltip

### ISSUE THREAD

**Issue by ascalamogna** [openaq/openaq.org#137]

**Comment by danielfdsilva** [comment]
@jflasher 

the map shows measurements from the `/latest` endpoint. This endpoint does not contain info about the sources (ie. `sourceName`) so showing source on the tooltip is kind of problematic...

you think we could add the `sourceName` to that endpoint?

**Comment by jflasher** [comment]
@danielfdsilva Do we already have access to `/locations` data? Could we use the location name to find the source in `/locations` and then get info in `/sources`? Seems a bit roundabout, but if we already get all the data, maybe alright?

**Comment by danielfdsilva** [comment]
@jflasher No, I don't think we do. And since the map lists all the locations, we'd need to download all the data from `/locations` and that would be way too much.

**Comment by jflasher** [comment]
@danielfdsilva ok, related to https://github.com/openaq/openaq.org/issues/112, I can potentially add `sourceNames: ['source 1', 'source2']` to `/latest`, that'd do it I think?

**Comment by jflasher** [comment]
On /latest, each measurement has its own source listed, since they may be different in some cases. Can probably just take a unique of them all and show those items on tooltip.
